.github/workflows/deploy.yml

```diff
@@ -1,60 +1,37 @@
 name: Deploy
 
 on:
-  push:
-    tags:
-    - '*.*.*'
+  workflow_run:
+    workflows: [Build]
+    branches: [master]
```

Comment:
if `deploy` runs only on `master`, how could we release an `rc` version?

Reply:
Good point, have added the `next` branch to the list of branches there too.